eng/Packages.props

```diff
@@ -37,7 +37,8 @@
     <PackageReference Update="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
     <PackageReference Update="System.Runtime.Loader" Version="4.0.0" />
     <PackageReference Update="System.Runtime.Serialization.Primitives" Version="4.1.1" />
-    <PackageReference Update="System.Security.Principal.Windows" Version="4.3.0" />
+    <PackageReference Update="System.Security.Permissions" Version="4.7.0" />
+    <PackageReference Update="System.Security.Principal.Windows" Version="4.7.0" />
```

Comment:
Taking the latest of System.Security.Permissions conflicted with the older version here so I bumped it. It's core-only so shouldn't impact VS.